spec/CampMinder/EstablishConnection_spec.rb

```diff
@@ -3,19 +3,18 @@
 describe CampMinder::EstablishConnection do
   before do
     @data = {
-      'clientID' => 'C-123',
-      'personID' => 'P-123',
+      'client_id' => 'C-123',
+      'person_id' => 'P-123',
       'token' => 'DEF-456',
-      'partnerClientID' => 'IEX-C-123'
+      'partner_client_id' => 'IEX-C-123'
     }
 
-    @payload = %{<?xml version="1.0" encoding="UTF-8"?>
-<responseObject>
-  <clientID>#{@data['clientID']}</clientID>
-  <personID>#{@data['personID']}</personID>
+    @payload = %{<connectionRequest version="1">
+  <clientID>#{@data['client_id']}</clientID>
+  <personID>#{@data['person_id']}</personID>
   <token>#{@data['token']}</token>
-  <partnerClientID>#{@data['partnerClientID']}</partnerClientID>
-</responseObject>
+  <partnerClientID>#{@data['partner_client_id']}</partnerClientID>
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.